docs/user-guide/kubectl-cheatsheet.md

```diff
@@ -118,16 +118,18 @@ $ kubectl get pods --sort-by=.status.containerStatuses[0].restartCount
 $ kubectl get pods --selector=app=cassandra rc -o 'jsonpath={.items[*].metadata.labels.version}'
 
 # Get ExternalIPs of all nodes
-$ kubectl get nodes -o jsonpath='{.items[*].status.addresses[?(@.type=ExternalIP)].address}'
+$ kubectl get nodes -o jsonpath='{.items[*].status.addresses[?(@.type=="ExternalIP")].address}'
 
 # List Names of Pods that belong to Particular RC
 # "jq" command useful for transformations that are too complex for jsonpath
 $ sel=$(./kubectl get rc <rc-name> --output=json | jq -j '.spec.selector | to_entries | .[] | "\(.key)=\(.value),"')
 $ sel=${sel%?} # Remove trailing comma
 $ pods=$(kubectl get pods --selector=$sel --output=jsonpath={.items..metadata.name})`
 
+# Print all node conditions
+$ kubectl get nodes -o jsonpath='{range .items[*]}{"\n"}{@.metadata.name}:{"\t"}{range @.status.conditions[*]}{" "}{@.type}={@.status}{end}{end}'
 # Check which nodes are ready
-$ kubectl get nodes -o jsonpath='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'| tr ';' "\n"  | grep "Ready=True" 
+$ kubectl get nodes -o jsonpath='{range .items[*]}{@.metadata.name}:{" "}{range @.status.conditions[?(@.type=="Ready")]}{@.status}{"\n"}{end}{end}' | grep True
```

Comment:
This didn't work correctly, so I've created an example above that does what original one did, and another to show output only the readiness.